ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/ServerInfo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client;
+
+public interface ServerInfo {
+
+  String getVersion();
+
+  String getKafkaClusterId();
+
+  String getKsqlServiceId();
+
+  String getServerStatus();
```

Comment:
This is just the command runner status, right? I don't think it's useful to Java client users, as in, I can't think of what someone would do in response to getting this information. I say let's leave it out for now in order to keep the interface simpler. We can always add it in later if there's demand for it.